geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/builder/SubFeatureTypeBuilder.scala

```diff
@@ -0,0 +1,15 @@
+package org.locationtech.geomesa.utils.geotools.builder
+import org.opengis.feature.`type`.{ComplexType,Name}
+
+class SubFeatureTypeBuilder[ReturnType](val target: DescriptorBuilder[ReturnType]) extends AbstractFeatureTypeBuilder[DescriptorBuilder[ReturnType]] {
+  override def namespace(ns: String) = super.namespace(ns)
+
+  def end = {
+    target.finish(makeType)

```

Comment:
For all these builders, I'd standardize on 'build'.  Unless there's a reason not to...